apps/marketplace/components/BuyerDashboard/BuyerDashboard.scss

```diff
@@ -1,58 +1,207 @@
-.container {
-  margin-bottom: 100px;
-}
-
 .header {
   margin-bottom: 40px;
 }
 
+.organisation {
+  opacity: 0.6;
+}
+
+ul.menu {
+  box-sizing: content-box;
+  list-style-type: none;
+  margin-left: 0;
+  margin-top: 1.5rem;
+  padding-left: 0;
+  border-bottom: 1px solid #bebebe;
+}
+
+ul.menu li {
+  display: inline-block;
+  margin: 0;
+}
+
+ul.menu li a {
+  display: block;
+  font-size: 1.25rem;
+  padding: 10px;
```

Comment:
should these all be `rem` ?

Reply:
Possibly - the nav/menu CSS was copied straight from the existing dashboard to try and get the same look and feel (as it's not a uitkit component anymore).